toponetx/algorithms/spectrum.py

```diff
@@ -255,7 +256,27 @@ def simplicial_complex_hodge_laplacian_spectrum(
     return laplacian_spectrum(SC.hodge_laplacian_matrix(rank=rank))
 
 
-def cell_complex_adjacency_spectrum(CC: CellComplex, rank):
+def path_complex_hodge_laplacian_spectrum(
+    PC: PathComplex, rank: int, weight: str = "weight"
```

Comment:
According to the docstring, `weight` is optional:
```suggestion
    PC: PathComplex, rank: int, weight: str | None = "weight"
```